fontforge/python.c

```diff
@@ -7887,12 +7935,20 @@ static PyGetSetDef PyFF_Glyph_getset[] = {
 /*  Glyph Methods  */
 /* ************************************************************************** */
 
-static PyObject *PyFFGlyph_Build(PyObject *self, PyObject *UNUSED(args)) {
+static PyObject *PyFFGlyph_Build(PyObject *self, PyObject *args) {
     SplineChar *sc = ((PyFF_Glyph *) self)->sc;
     int layer = ((PyFF_Glyph *) self)->layer;
+    int accent_hint = false;
+    PyObject *accent_hint_pyo;
+
+    if ( PyArg_ParseTuple(args, "|O", &accent_hint_pyo) ) {
```

Comment:
Why not just use `"|p"` and pass `accent_hint` directly into the parser?

Reply:
@jtanx I can't use `p` because it's "New in version 3.3". @frank-trampe said to _at least_ make sure this builds under 2.x, and that old scripts keep running under 2.x.